drivers/display/ili9340/Kconfig

```diff
@@ -0,0 +1,88 @@
+# Kconfig - ILI9340 display driver configuration options
+
+#
+# Copyright (c) 2017 dXplore
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ILI9340
+	bool "ILI9340 display driver"
+	depends on DISPLAY && SPI
+	default n
+	help
+	Enable driver for ILI9340 display driver.
+
+config ILI9340_DEV_NAME
+	string "ILI9340 device name"
+	depends on ILI9340
+	default "ILI9340"
+
+config ILI9340_SPI_DEV_NAME
+	string "SPI master where ILI9340 is connected"
+	depends on ILI9340
+	default "SPI_0"
+	help
+	Specify the device name of the I2C master device to which ILI9340 is
+	connected.
+
+config SYS_LOG_ILI9340_LEVEL
```

Comment:
That one you could generalize it in the main display Kconfig and call it SYS_LOG_DISPLAY_LEVEL